third_party/forked/libcontainer/utils/utils.go

```diff
@@ -0,0 +1,35 @@
+package utils
```

Comment:
is this intentionally unused now?

Reply:
E_TOO_MANY_BRANCHES... pushed an update now :)